CalendarApp/BookingRoom/Controllers/UpdateEventController.cs

```diff
@@ -1,26 +1,42 @@
-﻿using BookingRoom.Models.GoogleCalendar;
+﻿using System.Collections.Generic;
+using BookingRoom.Models.GoogleCalendar;
 using System.Net;
 using System.Net.Http;
 using System.Web.Http;
 using BookingRoom.Models.GoogleEvent;
 using GoogleCalendarService;
 using BookingRoom.Filters;
+using Google.Apis.Calendar.v3.Data;
+using GoogleCalendarService.Manager;
 
 namespace BookingRoom.Controllers
 {
     [GoogleExceptionFilter]
     public class UpdateEventController : BaseEventController
     {
-        public UpdateEventController(BookingService bookingService)
-            : base(bookingService)
+        public UpdateEventController(BookingService bookingService, EventManager manager)
+            : base(bookingService, manager)
         {
 
         }
         [HttpPut]
-        public HttpResponseMessage UpDateEvent(CalendarEvent eventForUpdate, string eventID)
+        public HttpResponseMessage UpdateEvent(CalendarEvent eventToUpdate, string eventID)
         {
-            BookingService.UpdateEvent(ToEventConverter.ToEvent(eventForUpdate), eventForUpdate.CalendarID, eventID);
-            return Request.CreateResponse(HttpStatusCode.Created, eventForUpdate);
+            string timeZone = BookingService.GetTimeZone(eventToUpdate.CalendarID);
+            Event eventToCheck = ToEventConverter.ToEvent(eventToUpdate, timeZone);
+            eventToCheck.Id = eventID;
+            List<Event> conflictedEvents = EventManager.CheckEvent(eventToUpdate.CalendarID, eventToCheck, eventToUpdate.ForcedEvent);
+            if (conflictedEvents.Count != 0)
+            {
+                var unSuccessfulRequest = new { Successful = false, ConflictedList = conflictedEvents };
+                return Request.CreateResponse(HttpStatusCode.OK, unSuccessfulRequest);
+            }
+            else
+            {
+                BookingService.UpdateEvent(ToEventConverter.ToEvent(eventToUpdate, timeZone), eventToUpdate.CalendarID, eventID);
+            }
+            var passRequest = new { Successful = true, ConflictedList = conflictedEvents };
```

Comment:
I vote for naming it  successful request to be consistent with unsuccessful
+ u don't need to return conflictedEvents  and you are not returning it in the previous method, make it consistent pls